nucliadb_texts/src/writer.rs

```diff
@@ -296,6 +296,26 @@ impl TextWriterService {
             self.schema.status => resource.status as u64,
         );
 
+        if self.schema.schema.get_field("groups_with_access").is_some() {
```

Comment:
I think this makes it possible to change the schema in a bw/compatible way.
I need to verify it though.

Reply:
nah, old versions of the index break on loading. we'll need another version of the texts library